src/modules/billing/services/supplementary-billing-service/data-service.js

```diff
@@ -46,6 +46,14 @@ const pickKeys = [
   'isNewLicence'
 ]
 
+const pick = (obj, props) => {
```

Comment:
I chose to add a function here for pick otherwise I would have had to duplicate the pickKeys which would have been an additional 25 lines of code

Reply:
I think you should be okay here. I can only see one reference to `pick()` and `pickKeys` which should mean you wouldn't need to repeat `pickKeys`. Happy to take a look with you!